### PR TITLE
Fix Prometheus endpoint

### PR DESCRIPTION
**Pull request by goruha**
Fix jaeger metrics port

The port was changed in this PR

https://github.com/jaegertracing/jaeger/issues/1332
https://github.com/jaegertracing/jaeger/pull/1442

image

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ x] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: install/kubernetes/helm/istio/charts/tracing/templates/deployment-jaeger.yaml [hunk 1]
```diff
@@ -24,11 +24,9 @@ spec:
       annotations:
         sidecar.istio.io/inject: "false"
         prometheus.io/scrape: "true"
-        prometheus.io/port: "16686"
+        prometheus.io/port: "14269"
 {{- if .Values.contextPath }}
         prometheus.io/path: "{{ .Values.contextPath }}/metrics"
-{{- else }}
```
Comment (morvencao) [inline install/kubernetes/helm/istio/charts/tracing/templates/deployment-jaeger.yaml]: why remove this, `contextPath` can be empty string.

**Comment by douglas-reid** [comment]
is this superseded by #17522 ?

**Comment by objectiser** [comment]
@douglas-reid yes

**Comment by douglas-reid** [comment]
@objectiser OK. thanks.

@goruha Thanks for this PR. Because #17522 will achieve the same result (in addition to other things), I'm going to close this PR.

**Pull request closed without merging**